src/prefect/orion/models/block_documents.py

```diff
@@ -211,7 +228,7 @@ async def _find_parent_block_document(
 
 @inject_db
 async def read_block_document_by_name(
-    session: sa.orm.Session,
+    session: AsyncSession,
```

Comment:
While I was in this module, I fixed these incorrect type hints.  We should get these corrected across the `models` package by and by.